osu.Game/Skinning/RulesetSkinProvidingContainer.cs

```diff
@@ -75,6 +86,12 @@ protected virtual void UpdateSkins()
                         break;
                 }
             }
+
+            var defaultSkinIndex = SkinSources.IndexOf(skinManager.DefaultSkin);
```

Comment:
```suggestion
            int defaultSkinIndex = SkinSources.IndexOf(skinManager.DefaultSkin);
```

Should probably also leave an inline comment explaining the scenarios where the default skin may not be present. Since I'm not immediately aware of where this is the case (tests..?)

Reply:
Yep tests may cache an `ISkinSource` that doesn't include the `SkinManager` skins in its `AllSources` property.